lib/commands/general.js

```diff
@@ -71,13 +71,27 @@ commands.background = async function (duration) {
 
 /*
  * Simulate Touch ID with either valid (match === true) or invalid (match === false)
- * fingerprint
+ * fingerprint (Simulator only)
  */
 commands.touchId = async function (match = true) {
-  let params = {
-    match
-  };
-  return await this.proxyCommand('/wda/touch_id', 'POST', params);
+  if (this.isSimulator()) {
+    return await this.proxyCommand('/wda/touch_id', 'POST', {match});
+  } else {
```

Comment:
I'd rather this be reversed, so we get out in the conditional, and then work outside of it.
```js
if (!this.isSimulator()) {
  throw new errors.UnknownError('Touch ID simulation not supported on real devices');
}
return await this.proxyCommand('/wda/touch_id', 'POST', {match});
```